gwlfe.py

```diff
@@ -0,0 +1,694 @@
+                        ET = KV[i] * PotenET * PcntET[i]
+
+                # Daily ET calculation
+                DailyET[y, i, j] = ET
+
+                # ***** END WEATHER DATA ANALYSIS *****
+
+                # ***** WATERSHED WATER BALANCE *****
+                # TODO: Placeholder declarations
+                RecessionCoef = 0
+                SatStor = 0
+                SeepCoef = 0
+                UnsatStor = 0
+                MaxWaterCap = 0
+                Perc = np.zeros((100, 100, 100))
+                PercCm = np.zeros((100, 100, 100))
+                DailyFlow = np.zeros((100, 100, 100))
+                DayRunoff = np.zeros((100, 100, 100))
+                DailyFlowGPM = np.zeros((100, 100, 100))
+                DailyGrFlow = np.zeros((100, 100, 100))
+                MonthFlow = np.zeros((100, 100))
+                Precipitation = np.zeros((100, 100))
+                Evapotrans = np.zeros((100, 100))
+                StreamFlow = np.zeros((100, 100))
+                GroundWatLE = np.zeros((100, 100))
+                NumPondSys = np.zeros(100)
+                NitrSepticLoad = 0
+                NitrPlantUptake = 0
+                Grow = np.zeros(100)
+                PhosSepticLoad = 0
+                PhosPlantUptake = 0
+                FrozenPondNitr = 0
+                FrozenPondPhos = 0
+
+                if QTotal <= Water:
+                    Infiltration = Water - QTotal
+                GrFlow = RecessionCoef * SatStor
+                DeepSeep = SeepCoef * SatStor
+
+                # CALCULATE EVAPOTRANSPIRATION, Percolation, AND THE
+                # NEXT DAY'S UNSATURATED STORAGE AS LIMITED BY THE UNSATURATED
+                # ZONE MAXIMUM WATER CAPACITY
+
+                UnsatStor = UnsatStor + Infiltration
+
+                # Calculate water balance for non-Pesticide componenets
+                if ET >= UnsatStor:
+                    ET = UnsatStor
+                    UnsatStor = 0
+                else:
+                    UnsatStor = UnsatStor - ET
+
+                # Obtain the Percolation, adjust precip and UnsatStor values
+                if UnsatStor > MaxWaterCap:
+                    Percolation = UnsatStor - MaxWaterCap
+                    Perc[y, i, j] = UnsatStor - MaxWaterCap
+                    UnsatStor = UnsatStor - Percolation
+                else:
+                    Percolation = 0
+                    Perc[y, i, j] = 0
+                PercCm[y, i, j] = Percolation / 100
+
+                # CALCULATE STORAGE IN SATURATED ZONES AND GROUNDWATER
+                # DISCHARGE
+                SatStor = SatStor + Percolation - GrFlow - DeepSeep
+                if SatStor < 0:
+                    SatStor = 0
+                Flow = QTotal + GrFlow
+                # DailyFlow(y, i, j) = Flow
+                DailyFlow[y, i, j] = DayRunoff[y, i, j] + GrFlow
+
+                DailyFlowGPM[y, i, j] = Flow * 0.00183528 * TotAreaMeters
```

Comment:
I brought over all of the comments from the main code block. Most of the arithmetic is undocumented.